tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -1,18 +1,27 @@
 ---
 title: "NServiceBus 101 Lesson 1: Getting started"
-reviewed: 2016-11-16
+reviewed: 2017-01-26
 ---
 
 In the next 10 minutes, you will learn how to set up a new development machine for NServiceBus and create your very first messaging endpoint.
 
 
-## Prerequisites
+## Before we get started
 
-There are a few things we need in order to build a solution with the latest version of NServiceBus.
+NServiceBus has very few prerequisites. It's very easy to get started.
```

Comment:
> It's very easy to get started

this is redundant. either it will be clear to the consumer that it is easy. or, if they struggle, this will make them feel stupid